packages/tldraw/src/lib/tools/SelectTool/childStates/PointingShape.ts

```diff
@@ -28,6 +29,13 @@ export class PointingShape extends StateNode {
 		this.hitShape = info.shape
 		const outermostSelectingShape = this.editor.getOutermostSelectableShape(info.shape)
 
+		// Bring sticky notes to front on pointer down;
+		// consider changing the logic to "move to front of any overlapping shapes"
+		// rather than move to front of all shapes in the page / parent
+		if (this.editor.isShapeOfType<TLNoteShape>(info.shape, 'note')) {
```

Comment:
let's make this an API property, not specific to `note`, something like `shouldBringToFrontOnSelect`

Reply:
I'd prefer we keep these hardcoded for now!